tasks/build.ts

```diff
@@ -43,4 +43,9 @@ c.API({
   outDir: "client/src/api",
   publicDir: "client/src/assets",
   watch,
+  getSitemapDestDir(config) {
```

Comment:
why not just `appDir: "client/src"`

Reply:
My thinking is it should be based on the configured dest path.